### PR TITLE
Split test_gamma_ray_grid into two new files in transport folder

### PR DESCRIPTION
**Pull request by nolanbrown01**
### :pencil: Description

**Type:** :roller_coaster: `infrastructure`

Split test_gamma_ray_grid.py into test_calculate_distances_3d.py and test_r_packet_transport_3d.py. Moved the tests associated with each file into the appropriate test file.

### :vertical_traffic_light: Testing

How did you test these changes?

- [x] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

**Comment by andrewfullard** [comment]
There are duplicated changes between this branch and #2078

**Comment by sonachitchyan** [comment]
I feel like the name ```packet``` for the object might be confusing in the future if we use a different procedure for ```r_packet``` and ```v_packet```?

File: tardis/transport/tests/test_r_packet_transport_3d.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    """
+
+    Parameters
+    ----------
+    basic_gamma_ray : GammaRay object
+    """
+    packet = basic_gamma_ray
+    distance = 1.0e15
+
+    new = packet.location + distance * packet.direction
```
Comment (andrewfullard) [inline tardis/transport/tests/test_r_packet_transport_3d.py:21]: This ought to be parametrized

**Comment by andrewfullard** [comment]
This is outdated

**Pull request closed without merging**